.github/workflows/test.yml

```diff
@@ -10,22 +10,25 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
       - name: Install Rust stable
-        uses: dtolnay/rust-toolchain@stable
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: stable
+          components: rustfmt
```

Comment:
This is a matter of taste, but you can combine the `@stable` with `components` just fine. No need to go for the `@master` here.
```
    uses: dtolnay/rust-toolchain@stable
    with:
        components: rustfmt
```

Reply:
Since the ones at the bottom required being explicit, I decided to make them all such so they were uniform. I definitely feel like it makes it easier to process when looking at the entire file.

Reply:
Ack, just thought I'd mention it.